### PR TITLE
Move SIPs to Stacksgov

### PR DESCRIPTION
**Pull request by jcnelson**
This PR moves all SIPs to their new home at https://github.com/stacksgov/sips.

### CODE REVIEW

File: sip/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Stacks Improvement Proposals
+
+This directory formally contained all of the in-progress Stacks Improvement
```
Comment (kantai) [inline sip/README.md:3]: Typo: formally -> formerly

**Review by diwakergupta — CHANGES_REQUESTED** [review]
If you've already copied over the SIPs, can we do this post launch?

A lot of places link to SIPs directly (esp. SIP-007) -- instead of deleting, my preference would be to just replace their text with a link to the new corresponding SIP location. After a few weeks of that we can delete the SIPs from here.

**Review by lgalabru — COMMENTED** [review]
If we already have the new links, should we also update the  `README.md` for this repo?

**Comment by jcnelson** [comment]
TODO: Add links to the new SIPs where the old SIPs are.

**Pull request closed without merging**